ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -82,6 +82,17 @@ const styles = reactStyles({
     background: 'transparent',
     cursor: 'pointer'
   },
+  error: {
+    background: colors.warning,
+    color: colors.white,
+    fontSize: '12px',
+    fontWeight: 500,
+    textAlign: 'left',
+    border: '1px solid #ebafa6',
+    borderRadius: '5px',
+    marginTop: '0.25rem',
+    padding: '8px',
+  },
```

Comment:
Should we make an generic error component that is reusable across all react components for error messaging?

Reply:
+1 to this – this PR caught my eye, as I was running into some scenarios elsewhere in the Workbench UI where we don't have a clear / consistent way of displaying errors. A core component here would be great.

Reply:
Agreed, we're already using this same template in cohort builder, cohort review and now concept search (maybe other places as well?).  I've filed a [ticket](https://precisionmedicineinitiative.atlassian.net/browse/RW-4406) for this and can follow-up on it.